methods/bluetoquenfe.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "bluetoquenfe",
+    "status": "registered",
+    "verifiableDataRegistry": "Trusted Digital Web",
+    "contactName": "Hyperonomy Digital Identity Lab, Parallelspace Corporation",
+    "contactEmail": "mwherman@parallelspace.net",
```

Comment:
Do you intend for this to be effectively the same as a change controller in which case only you @mwherman2000 are allowed to make updates to this entry?